scripts/docker/cleanimages_mac.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+if [[ $1 == "" ]]; then
+    echo "arg1 - Path to CSV file with project,sha"
+    exit
+fi
+
+projfile=$1
+
+for line in $(cat ${projfile}); do
+    # Create name of image for each project, then delete it
+    slug=$(echo ${line} | cut -d',' -f1 | rev | cut -d'/' -f1-2 | rev)
+    modifiedslug=$(echo ${slug} | gsed 's;/;.;' | tr '[:upper:]' '[:lower:]')
```

Comment:
Is literally the only difference between these Mac version of the scripts with the original ones the change from "sed" to "gsed"? Is everything else exactly the same?

Reply:
Yeah that's the only thing that changed, maybe there is a better way so we only need 1 set of scripts?

Reply:
Yes, it would probably be best to have only one set of scripts, since if we need to change other logic we would have to remember to propagate such changes to these separate Mac scripts as well.

Forgive my ignorance of Mac, but what's the difference between sed and gsed here? Is it guaranteed that a Mac environment would have gsed? Is it enough to have logic to just check if gsed exists and run gsed in that case? Or to check what the operating system is and if Mac then use gsed?

Reply:
It is not guaranteed that a Mac environment has gsed (at least mine doesn't). Quick searching on Google suggests that depending on how one installs sed on Mac, you may have to type gsed or sed.
https://stackoverflow.com/questions/30003570/how-to-use-gnu-sed-on-mac-os-10-10-brew-install-default-names-no-longer-su

@henryw30  can you try reinstalling your sed to see if this gsed change is really necessary? If it is not, we can still add a FAQ section that describes this issue of sed vs gsed.